plugin/trino-oracle/src/test/java/io/trino/plugin/oracle/BaseOracleConnectorTest.java

```diff
@@ -90,11 +90,8 @@ protected Optional<DataMappingTestSetup> filterDataMappingSmokeTestData(DataMapp
     {
         String typeName = dataMappingTestSetup.getTrinoTypeName();
         if (typeName.equals("date")) {
-            // TODO (https://github.com/trinodb/trino/issues) Oracle connector stores wrong result when the date value <= 1582-10-14
-            if (dataMappingTestSetup.getSampleValueLiteral().equals("DATE '0001-01-01'")
-                    || dataMappingTestSetup.getSampleValueLiteral().equals("DATE '1582-10-04'")
-                    || dataMappingTestSetup.getSampleValueLiteral().equals("DATE '1582-10-05'")
-                    || dataMappingTestSetup.getSampleValueLiteral().equals("DATE '1582-10-14'")) {
+            // Oracle TO_DATE function returns +10 days during julian and gregorian calendar switch
```

Comment:
Can you leave a code comment why '1582-10-14' passes? If only '1582-10-05' is a matter and other days of calendar switch are fine, this comment should be updated.

Reply:
Currently, seems no test cases for the '1582-10-14'(sample value)

Reply:
Right, but it exists as highValueLiteral.